src/test/isolation2/input/resgroup/resgroup_startup_memory.source

```diff
@@ -0,0 +1,105 @@
+-- start_ignore
+drop table if exists t1;
+drop role if exists test1;
+drop role if exists test2;
+drop resource group rg1;
+drop resource group rg2;
+drop resource group rg3;
+-- end_ignore
+
+create extension if not exists gp_inject_fault;
+
+create or replace function resGroupPalloc(float) returns int as '@abs_builddir@/../regress/regress@DLSUFFIX@', 'resGroupPalloc' language C reads sql data;
+create or replace function hold_memory_by_percent_on_qe(float) returns int as $$
+select resGroupPalloc($1) from gp_dist_random('gp_id')
+$$ language sql;
+
+create or replace function repeatPalloc(int, int) returns int as
+'@abs_builddir@/../regress/regress@DLSUFFIX@', 'repeatPalloc'
+    language C reads sql data;
+
+create or replace function hold_memory(int) returns int as $$
+select * from repeatPalloc(1, $1)
+$$ language sql;
```

Comment:
can we replace `hold_memory` with `repeatPalloc(40, 1)`?

Reply:
done